functions/go/apply-replacements/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"os"
+
+	sdk "github.com/GoogleContainerTools/kpt-functions-catalog/thirdparty/kyaml/fnsdk"
+	"sigs.k8s.io/kustomize/kyaml/fn/framework"
+)
+
+func main() {
+	if err := sdk.AsMain(sdk.ResourceListProcessorFunc(applyReplacements)); err != nil {
+		os.Exit(1)
+	}
+}
+
+func applyReplacements(rl *sdk.ResourceList) error {
+	r := Replacements{}
+	return r.Process(&framework.ResourceList{
```

Comment:
Since this KRM function is using the fnsdk library, can you use the [fnsdk.ResourceList](https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/master/thirdparty/kyaml/fnsdk/resourcelist.go#L23)?

Reply:
+1

Reply:
done!